CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 This file is used to list changes made in each version of the haproxy cookbook.
 
+## [v8.1.2] (2019-10-10)
+
+ - Removed `.rubocop.yml` as no longer required
```

Comment:
Hey,

thanks for the Changelog entry, really appreciated, could you indent it at the start, that will fix the build errors in Markdown Lint
```suggestion
- Removed `.rubocop.yml` as no longer required
```

Reply:
Done